src/php/cloud/class-cloud-search-list-table.php

```diff
@@ -163,14 +163,12 @@ public function display_rows() {
 						<p class="authors">
 							<cite>
 								<?php
-								esc_html_e( 'Codevault: ', 'code-snippets' );
-
 								printf(
-									'<a target="_blank" href="%s">%s</a>',
+									'%s <a target="_blank" href="%s">%s</a>',
+									esc_html__( 'Codevault:', 'code-snippets' );
```

Comment:
Prevent this:

![image](https://github.com/user-attachments/assets/ededcbfd-72ba-43a0-a21e-b5e1d6fdde1e)